lib/generators/fuel/views_generator.rb

```diff
@@ -9,7 +9,7 @@ class ViewsGenerator < Rails::Generators::Base
       source_root File.expand_path('../../../../app/views', __FILE__)
 
       def copy_views
-        directory 'fuel', 'app/views/fuel'
+        directory 'fuel', 'app/views/fuel/posts'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.